### PR TITLE
Fix issue on starting mongodb replicaset

### PR DESCRIPTION
**Pull request by llaske**
On recent version of Kubernetes (1.18.x), the mongodb pods don't start. In the event log a message mention:

`Warning   FailedCreate            statefulset/mymongodb-mongodb-replicaset                create Pod mymongodb-mongodb-replicaset-0 in StatefulSet mymongodb-mongodb-replicaset failed error: Pod "mymongodb-mongodb-replicaset-0" is invalid: spec.containers[0].startupProbe.successThreshold: Invalid value: 2: must be 1`

The default the `successThreshold` value should be 1 and it's 2 by default.

Plus the `https://kubernetes-charts.storage.googleapis.com/` repo no longer exist, `mymongodb` is now available directly in `https://charts.helm.sh/stable` repo.

### CODE REVIEW

**Review by NikhilM98 — CHANGES_REQUESTED** [review]
@llaske Thanks for the fix!

The yaml file does not requires all the default values.
It would be sufficient if you only add:
```
startupProbe:
  successThreshold: 1
```

In the `mongodb-values.yaml` file.

**Comment by llaske** [comment]
You're right. Fixed.

File: scripts/charts/mongodb.sh [hunk 1]
```diff
@@ -13,8 +13,6 @@ NC='\033[0m';
 printf "${YELLOW}Checking for MongoDB-Replicaset with releasename: ${BLUE}mymongodb\n${NC}";
 helm status mymongodb >/dev/null 2>&1 || {
     printf >&2 "${BLUE}Chart not found. Installing MongoDB-Replicaset...\n${NC}";
-    helm repo add stable https://kubernetes-charts.storage.googleapis.com/;
```
Comment (NikhilM98) [inline scripts/charts/mongodb.sh:16]: You should not remove the `helm repo add` command.

**Review by NikhilM98 — CHANGES_REQUESTED** [review]

File: scripts/charts/mongodb.sh (hunk 1 above)
Comment (llaske) [reply scripts/charts/mongodb.sh:16]: The `https://kubernetes-charts.storage.googleapis.com/` repo no longer exist (access denied, see [here](https://stackoverflow.com/questions/61954440/how-to-resolve-https-kubernetes-charts-storage-googleapis-com-is-not-a-valid)), `mymongodb` is now available directly in `https://charts.helm.sh/stable` repo.

Comment (NikhilM98) [reply scripts/charts/mongodb.sh:16]: Oh. Thanks for the explanation @llaske

**Pull request merged**